src/pages/organize/[orgId]/settings/index.tsx

```diff
@@ -49,15 +47,18 @@ const SettingsPage: PageWithLayout = () => {
           <Msg id={messageIds.administrators.title} />
         </Typography>
         <RoleAddPersonButton
-          disabledList={adminRoles}
+          disabledList={listFuture.filter((user) => user.role === 'admin')}
           orgId={orgId}
           roleType="admin"
         />
       </Box>
       <Typography mb={2} variant="body2">
         <Msg id={messageIds.administrators.description} />
       </Typography>
-      <OfficialList officialList={adminRoles} orgId={orgId} />
+      <OfficialList
+        officialList={listFuture.filter((user) => user.role === 'admin')}
```

Comment:
I guess we can make assign this in `const` since it is used in another place! so does role ==='organizer'!